pinyin.go

```diff
@@ -201,3 +201,21 @@ func Slug(s string, a Args) string {
 	separator := a.Separator
 	return strings.Join(LazyPinyin(s, a), separator)
 }
+
+//返回汉字首字母,并且保留输入文字中的非汉字, 可以选择是否强制输出全小写字母
+func SimplePinyin(s string, isLowercase bool) string {
```

Comment:
暂时没有想到

mozillazg <notifications@github.com>于2016年4月29日 周五下午10:34写道：

> In pinyin.go
> <https://github.com/mozillazg/go-pinyin/pull/7#discussion_r61586909>:
>
> > @@ -201,3 +201,21 @@ func Slug(s string, a Args) string {
> >  	separator := a.Separator
> >  	return strings.Join(LazyPinyin(s, a), separator)
> >  }
> > +
> > +//返回汉字首字母,并且保留输入文字中的非汉字, 可以选择是否强制输出全小写字母
> > +func SimplePinyin(s string, isLowercase bool) string {
>
> 不知道是否有更贴切的名字来命名这个功能？
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mozillazg/go-pinyin/pull/7/files/c84ad6ebda83e756907d060307138ba170789068#r61586909>
>